airbyte-cdk/bulk/toolkits/load-object-storage/src/main/kotlin/io/airbyte/cdk/load/pipline/object_storage/ObjectLoaderPartFormatterStep.kt

```diff
@@ -25,8 +24,6 @@ class ObjectLoaderPartFormatterStep(
     private val stepId: String,
     private val flushStrategy: PipelineFlushStrategy?,
 ) : LoadPipelineStep {
-    override val numWorkers: Int = objectLoader.numPartWorkers
```

Comment:
why is this ever different from numWorkers?

Reply:
files just have 1 input partition so just need 1 worker